uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -34,18 +34,19 @@ <h4 class="modal-title">
       </ui-select>
     </div>
     <div
-      ng-class="{'chr_disabled': !$ctrl.request.contact_id}"
+      ng-class="{'chr_disabled': (!$ctrl.request.contact_id || $ctrl.noEntitlement)}"
```

Comment:
Code has changed to `$ctrl.absenceTypes.length === 0` but I am disabling only once.